pages/creator.vue

```diff
@@ -1,8 +1,45 @@
 <template class="z-0">
   <div>
     <HeaderNav />
-    <HeroSection />
-    
+    <HeroCreator></HeroCreator>
+    <HowItWorksCreator></HowItWorksCreator>
+    <!-- CARROUSEL -->
+    <GetInvolved class=" mb-32"></GetInvolved>
+
+    <!-- 2 MILLON -->
+    <div class="relative">
+      <div class="flex justify-center items-center">
+        <div class="relative flex items-center w-full max-w-screen-xl h-[422px] lg:h-[570px] p-8 lg:p-0">
+          <div class="relative flex flex-col justify-center items-center p-8 bg-[#2565F4] rounded-[20px] mx-auto">
+            <h1 class="text-5xl lg:text-[180px]  font-extrabold text-white text-center">2+ Million</h1>
+            <p class=" text-3xl lg:text-[60px] font-medium text-white text-center mt-8">Total paid out to creators</p>
+          </div>
+        </div>
+      </div>
+      <div class="absolute max-h-[422px] lg:max-h-full overflow-hidden inset-0 flex items-center justify-center text-[72px] font-medium leading-none z-[-1]">
+        <div class="w-full h-full text-center">
+          <p v-for="n in 8" :key="n">
+            <span class="gradient-text">{{"$".repeat(100)}}</span>
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <FeaturesCreator></FeaturesCreator>
+    <MissionDriven></MissionDriven>
+    <TestimonialsCreator></TestimonialsCreator>
+    <OrganizationCarousel />
     <Footer />
```

Comment:
This footer looks like this:

<img width="1165" alt="Screenshot 2024-07-29 at 6 12 46 PM" src="https://github.com/user-attachments/assets/00970dca-76e7-4928-b510-62b3c2e9227e">

Which is weird for a creator landing page. This is the design:


<img width="877" alt="Screenshot 2024-07-29 at 6 13 15 PM" src="https://github.com/user-attachments/assets/29753f4e-0294-4eda-8029-c55b3cc01bfb">

https://www.figma.com/design/stbe6XAd5jo5vTjzmZ0ZlG/Desktop-%26-Mobile-UI?node-id=22055-4532&m=dev